src/ipc/Coordinator.cc

```diff
@@ -143,6 +148,21 @@ void Ipc::Coordinator::handleRegistrationRequest(const HereIamMessage& msg)
     SendMessage(MakeAddr(strandAddrLabel, msg.strand.kidId), message);
 }
 
+void
+Ipc::Coordinator::handleForegroundRebuildMessage(const StrandMessage& msg)
+{
+    // notify any searchers waiting for this strand
+    for (const auto searchRequest: searchers) {
```

Comment:
Let's avoid copying Strings.
```suggestion
    for (const auto &searchRequest: searchers) {
```